graphx/src/main/scala/org/apache/spark/graphx/Pregel.scala

```diff
@@ -109,19 +109,17 @@ object Pregel extends Logging {
    * @return the resulting graph at the end of the computation
    *
    */
-  def apply[VD: ClassTag, ED: ClassTag, A: ClassTag]
-     (graph: Graph[VD, ED],
+  def apply[VD: ClassTag, ED: ClassTag, A: ClassTag](
```

Comment:
Although it's really minor, the paren placement now makes it hard to see that there are two arg lists. I think it was better as-is.